go_api/go/src/wallarooapi/serialize.go

```diff
@@ -81,7 +76,8 @@ func ComponentSerializeGetSpaceWrapper(componentId uint64, componentTypeId uint6
 
 //export ComponentSerializeWrapper
 func ComponentSerializeWrapper(componentId uint64, p unsafe.Pointer, componentType uint64) {
-	buff := serializedDicts[componentType].get(componentId)
+	result := serializedDicts[componentType].get(componentId)
+	buff := result.([]byte)
```

Comment:
i thinkt he buffer conversion should be in get instead of here. makes everything a little more clear.

Reply:
will update